src/handlers/comments/putComment.js

```diff
@@ -3,7 +3,13 @@ const nodemailer = require('nodemailer');
 const config = require('../../../config.json');
 const { stripTags } = require('../../util/functions');
 
+const successMessage =
+    'Successfully added comment. It will need to be accepted by an administrator before it is published.';
+
 async function putComment(request, h) {
+    // Silently drop ridiculously short comments, which are pretty much guaranteed to be spam.
+    if (request.payload.message?.length < 5) return { message: successMessage };
```

Comment:
This create a timing-based side-channel, but we don't really care. :D

Reply:
If we didn't want people to know that we drop very short comments, I feel like publishing the source would be the bigger problem. :P But, indeed we don't.